main.tf

```diff
@@ -157,6 +157,8 @@ data "google_container_engine_versions" "region" {
 }
 
 data "google_container_engine_versions" "zone" {
+  provider = "google-beta"
```

Comment:
This change is not relevant to the branch topic.

Reply:
This appears to be the result of a code skew between the autogen code and the post autogen'd code introduced by https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/commit/1d73e9d56ee0026aa09ddc4f8e6b2a2c325c1c93

tl;dr: I screwed up in a commit outside of this branch so regenerating the documentation makes a diff in the generated code unavoidable.

That being said, the diff should be for removing the provider definition from the `autogen/main.tf` file, not adding it here.

This should now be fixed.